### PR TITLE
[WIP] Issue #1894

### PR DESCRIPTION
**Pull request by yatsukhnenko**
Add Redis::lMove command.

### CODE REVIEW

File: redis_commands.c [hunk 1]
```diff
@@ -2139,6 +2139,39 @@ int redis_setbit_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
...
+            "Wherefrom argument must be either 'LEFT' or 'RIGHT'");
+        return FAILURE;
+    } else if (strcasecmp(to, "left") != 0 && strcasecmp(to, "right") != 0) {
+        php_error_docref(NULL, E_WARNING,
+            "Whereto argument must be either 'LEFT' or 'RIGHT'");
+        return FAILURE;
+    }
+
+    /* Construct command */
+    *cmd_len = REDIS_CMD_SPPRINTF(cmd, "LMOVE", "kkss",
```
Comment (michael-grunder) [inline redis_commands.c:2168]: Should this be "kkvv" since we serialize list elements?

**Review by michael-grunder — CHANGES_REQUESTED** [review]
Looks good.  I think we just need to serialize values

File: redis_commands.c (hunk 1 above)
Comment (yatsukhnenko) [reply redis_commands.c:2168]: There are no list elements in this command 😄

Comment (michael-grunder) [reply redis_commands.c:2168]: 😂  I should have looked up the command.

**Review by michael-grunder — APPROVED** [review]

**Pull request merged**